scripts/generate-rule-files/index.ts

```diff
@@ -0,0 +1,139 @@
+import { pascalCase } from 'change-case';
+import * as fs from 'fs';
+import { join } from 'path';
```

Comment:
Please use `node:fs` and `node:path`, it's safe that this script runs always on node v16+